src/properties/output/index.test.js

```diff
@@ -1,14 +1,103 @@
 import schema from './index'
 import { allValid, allInvalid } from '../../../test/utils'
+import { notAbsolutePath, absolutePath, urlPart } from '../../types'
 
 const validModuleConfigs = [
   // #0
-  // 'entry.js',
+  { filename: 'foo' },
+  // #1
+  { chunkFilename: 'foo' },
+  // #2
+  { path: 'foo/bar' },
+  // #3
+  { path: '/foo/bar' },
+  // #4
+  { publicPath: '/assets/' },
+  // #5
+  { publicPath: 'http://cdn.example.com/assets/[hash]/' },
+  // #6
+  { devtoolModuleFilenameTemplate: 'webpack:///[resourcePath]?[hash]' },
+  // #7
+  { devtoolModuleFilenameTemplate: () => {} },
+  // #8
+  { hotUpdateChunkFilename: '[id].[hash].hot-update.js' },
+  // #9
+  { hotUpdateMainFilename: '[hash].hot-update.json' },
+  // #10
+  { jsonpFunction: 'webpackJsonp' },
+  // #11
+  { hotUpdateFunction: 'webpackHotUpdate' },
+  // #12
+  { pathinfo: false },
+  // #13
+  { library: 'redux' },
+  // #14
+  { libraryTarget: 'commonjs' },
+  // #15
+  { crossOriginLoading: false },
+  // #16
+  { crossOriginLoading: 'anonymous' },
+
 ]
 
 const invalidModuleConfigs = [
   // #0
-  // { input: 1, error: { message: '"value" must be a string' } },
+  {
```

Comment:
Sure, i just didn't because that's a joi default, so I was not even thinking about it. Should probably be made explicit via tests anyway.